### ISSUE TITLE
unicycler

### ISSUE THREAD

**Issue by yolanda-huang** [ubccr/software-layer#169]
https://github.com/rrwick/Unicycler

**Comment by tonykew** [comment]
Test build completed - more info to follow

**Comment by tonykew** [comment]
unicycler version 0.5.0 has been published to the ccrsoft/2023.01 software release


```
vortex1$ module spider unicycler
----------------------------------------------------------------------------
  unicycler: unicycler/0.5.0
----------------------------------------------------------------------------
    Description:
      Unicycler is an assembly pipeline for bacterial genomes. It can
      assemble Illumina-only read sets where it functions as a
      SPAdes-optimiser. It can also assembly long-read-only sets (PacBio or
      Nanopore) where it runs a miniasm+Racon pipeline. 


    You will need to load all module(s) on any one of the lines below before the "unicycler/0.5.0" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
[...]
vortex1$ module load gcc/11.2.0 openmpi/4.1.1 unicycler/0.5.0
vortex1$ unicycler --help
usage: unicycler [-h] [--help_all] [--version] [-1 SHORT1] [-2 SHORT2]
                 [-s UNPAIRED] [-l LONG] -o OUT [--verbosity VERBOSITY]
                 [--min_fasta_length MIN_FASTA_LENGTH] [--keep KEEP]
                 [-t THREADS] [--mode {conservative,normal,bold}]
                 [--linear_seqs LINEAR_SEQS]
[...]
vortex1$ 
```
